README.md

```
@@ -8,22 +8,51 @@ My personal dotfiles managed with [nix-darwin] and [home-manager].
+| `nix`       | Install Nix (official installer)                        |
+| `homebrew`  | Install Homebrew                                        |
+| `taps`      | Tap and trust the third-party Homebrew taps             |
+| `etc`       | Move aside the `/etc` files nix-darwin wants to own     |
+| `backups`   | Remove stale `*.before-nix-darwin` symlink backups      |
+| `activate`  | Run the nix-darwin switch                               |
+| `zsh-cache` | Clear the stale zsh profile cache                       |
+
+Every step is idempotent, so re-running is safe:
+
+```console
+$ ./bootstrap.sh                 # run every step in order
```

markdownlint-cli2 (0.23.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)